ios/RNKustomerModule/RNKustomerModule.m

```diff
@@ -36,7 +36,7 @@ @implementation RNKustomerModule
 RCT_EXPORT_METHOD(identify:(NSString *) hash)
 {
   dispatch_async(dispatch_get_main_queue(), ^{
-    [Kustomer identify:hash];
+    [Kustomer identify:hash callback: nil];
```

Comment:
@cantide5ga don't know enough about Obj-C, is nil correct to use here?  Obv we'll want to implement that callback at some point whenever we want to start using `identify`

Reply:
I expect not, though I think this was the right first step so it compiles